src/app_service/service/general/service.nim

```diff
@@ -142,11 +143,12 @@ QtObject:
       return e.msg
 
   proc asyncImportLocalBackupFile*(self: Service, filePath: string) =
+    let formattedFilePath = singletonInstance.utils.fromPathUri(filePath)
```

Comment:
It would be good to have utilities like that `fromPathUri` as an independent standalone function, with no need to accessing it via some singleton instances.